FaderPlugin/Windows/Config/ConfigWindow.Settings.cs

```diff
@@ -177,160 +275,186 @@ private void Settings()
                     }
 
-                    // Down
                     ImGui.SameLine();
-                    if(ImGui.Button($"{FontAwesomeIcon.ArrowDown.ToIconString()}##{elementName}-{i}-down"))
+                    if (ImGui.Button($"{FontAwesomeIcon.ArrowDown.ToIconString()}##{elementName}-{i}-down"))
                     {
-                        if(i < SelectedConfig.Count - 1)
+                        if (i < SelectedConfig.Count - 1)
                         {
                             var swap1 = SelectedConfig[i + 1];
                             var swap2 = SelectedConfig[i];
-
-                            if(swap1.state != State.Default && swap2.state != State.Default)
+                            if (swap1.state != State.Default && swap2.state != State.Default)
                             {
                                 SelectedConfig[i] = swap1;
                                 SelectedConfig[i + 1] = swap2;
-
                                 SaveSelectedElementsConfig();
                             }
                         }
                     }
 
-                    // Delete
                     ImGui.SameLine();
-                    if(ImGui.Button($"{FontAwesomeIcon.TrashAlt.ToIconString()}##{elementName}-{i}-delete"))
+                    if (ImGui.Button($"{FontAwesomeIcon.TrashAlt.ToIconString()}##{elementName}-{i}-delete"))
                     {
                         SelectedConfig.RemoveAt(i);
-
                         SaveSelectedElementsConfig();
                     }
                 }
+            }
 
-                using var font = ImRaii.PushFont(UiBuilder.IconFont);
-                if(ImGui.Button($"{FontAwesomeIcon.Plus.ToIconString()}##{elementName}-add"))
+            // Add new condition row
+            ImGui.SameLine();
+            using (var font = ImRaii.PushFont(UiBuilder.IconFont)){;
+                if (ImGui.Button($"{FontAwesomeIcon.Plus.ToIconString()}##{elementName}-add"))
                 {
-                    // Add the new state then swap it with the existing default state.
                     SelectedConfig.Add(new ConfigEntry(State.None, Setting.Hide));
                     var swap1 = SelectedConfig[^1];
                     var swap2 = SelectedConfig[^2];
-
                     SelectedConfig[^2] = swap1;
                     SelectedConfig[^1] = swap2;
-
                     SaveSelectedElementsConfig();
                 }
             }
+            // Warning Label
+            {
+                var defaultEntry = SelectedConfig.FirstOrDefault(e => e.state == State.Default);
+                bool defaultDisabled = defaultEntry != null && defaultEntry.setting == Setting.Hide;
+                bool hoverPresent = SelectedConfig.Any(e => e.state == State.Hover);
+
+                if (defaultDisabled && hoverPresent)
+                {
+                    ImGui.Separator();
+                    ImGui.TextColored(ImGuiColors.DalamudRed, Language.StateWarning);
```

Comment:
Please use the `Helper.` version.
If Fader doesn't include it yet, take it from here https://github.com/Infiziert90/PlayerTrack/blob/master/PlayerTrack.Plugin/Windows/Helper.cs#L23